cmd/ansible.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"log"
+
+	"github.com/spf13/cobra"
+	analyzer "github.com/ukinimod/iac-count/pkg/analyzer"
+	"github.com/ukinimod/iac-count/pkg/core"
+	"github.com/ukinimod/iac-count/pkg/output"
+)
+
+func getValidMetrics() []string {
+	var metricList []string
+	for _, metric := range MetricList {
+		if core.IsValidMetric(metric) {
+			metricList = append(metricList, metric)
+		} else {
+			log.Printf("[WARN] Metric %s is not a valid metric, see `describe` command for the list of valid metrics.\n", metric)
+		}
+	}
+
+	return metricList
+}
+
+var CmdAnsible = &cobra.Command{
+	Use:   "ansible [root of ansible project]",
```

Comment:
In shell command helps a square bracket universially mean optional parameters. But in this case the parameter is mandatory.

Something like this would be more appropriate: 

```suggestion
	Use:   "ansible ANSIBLE_ROOT",
```